pkg/provider-local/webhook/prometheus/mutator.go

```diff
@@ -57,9 +57,8 @@ func (m *mutator) Mutate(_ context.Context, newObj, _ client.Object) error {
 				},
 				{
 					SourceLabels: []monitoringv1.LabelName{"__name__"},
-					// TODO: add filter for relevant metrics
-					Regex:  "^(up|task_delay_seconds|task_duration_seconds|total_duration_seconds|task_result_total)$",
-					Action: "keep",
+					Regex:        "(up|flow_.+)",
```

Comment:
Unfortunately, I still can't find the `flow` metrics in the performance prometheus. Can you dig deeper into why that is?

Reply:
The `metrics` package containing the `init()` function was not imported by any other package (anymore).